blueman/plugins/applet/DhcpClient.py

```diff
@@ -57,7 +57,7 @@ def err(_obj: Mechanism, result: GLib.Error, _user_data: None) -> None:
 
                 self.quering.remove(device)
 
-            Notification(_("Bluetooth Network"), _("Trying to obtain an IP address on %s\nPlease wait…" % device),
+            Notification(_("Bluetooth Network"), _(f"Trying to obtain an IP address on {device}\nPlease wait…"),
```

Comment:
https://github.com/blueman-project/blueman/pull/1306#issuecomment-665910288 still applies, doesn't it?